figures/backfill.py

```diff
@@ -0,0 +1,55 @@
+"""Backfills metrics models
+
+Initially developed to support API performance improvements
+"""
+
+from datetime import datetime
+from dateutil.rrule import rrule, MONTHLY
+
+from django.utils.timezone import utc
+
+from figures.mau import get_mau_from_student_modules
+from figures.models import SiteMonthlyMetrics
+from figures.sites import get_student_modules_for_site
+
+
+def backfill_monthly_metrics_for_site(site, overwrite):
+    """Quick hack function to backfill all historical site metrics for the site
+
+    We are a bit verbose with the output to help testing and validation since
+    this function was quickly put together
+    """
+    site_sm = get_student_modules_for_site(site)
+    first_created = site_sm.order_by('created').first().created
+
+    # We do this because there _might_ be a bug in rrule. It was skipping over
```

Comment:
```suggestion
    # We do this because there _might_ be a bug in rule. It was skipping over
```

Reply:
@melvinsoft Actually  it is "rrule": https://dateutil.readthedocs.io/en/stable/rrule.html

I'll make it clear that it is "dateutil rule"

Reply:
Yup, I had to look that up too. It's short for Reoccurence Rules

Reply:
Updated